src/huggingface_hub/keras_mixin.py

```diff
@@ -97,6 +99,9 @@ def push_to_hub_keras(
             Specify this in case you want to push to an existing repository in the hub. If unspecified, a new
             repository will be created in your namespace (unless you specify an :obj:`organization`) with
             :obj:`repo_name`.
+        tensorboard_dir (:obj:`str`, `optional`):
```

Comment:
I feel `log_dir` might be a nicer name so we're consistent both with `tensorflow` and `transformers`. WDYT?